clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/BackgroundEvent.java

```diff
@@ -38,6 +38,12 @@ public enum Type {
      */
     private final Uuid id;
 
+    /**
+     * The time in milliseconds when this event was enqueued.
+     * This field can be changed after the event is created, so it should not be used in hashCode, equals, or toStringBase.
+     */
+    private long enqueuedMs;
```

Comment:
We also need the `enqueuedMs` in the `toStringBase()` method as per the `ApplicationEvent`’s method of the same name.